lib/Controller/Attribute/ResponseHeader.php

```diff
@@ -28,6 +26,66 @@
  *
  */
 #[\Attribute(\Attribute::IS_REPEATABLE | \Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
-final class ResponseHeader extends BaseResponseHeader
+final class ResponseHeader
 {
+    protected string $key;
+
+    protected string|array $values;
+
+    protected bool $replace = false;
+
+    /**
+     * @param string|array|null $key
+     * @param string|array $values
+     * @param bool $replace
+     */
+    public function __construct($key = null, $values = '', $replace = false)
+    {
+        if (is_array($key)) {
+            // value is the default key if attribute was called without assignment
+            // e.g. #[ResponseHeader("X-Foo")] instead of #[ResponseHeader(key="X-Foo")]
+            if (isset($key['value'])) {
+                $key['key'] = $key['value'];
+                unset($key['value']);
+            }
+        } else {
+            $this->key = $key;
```

Comment:
`$key` can be `null`, so it will throw here an exception. Should we remove the `null` type hint and remove the default value?